libse/SubtitleFormats/UnknownSubtitle4.cs

```diff
@@ -139,5 +125,37 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
             }
             subtitle.Renumber();
         }
+
+        public string DecodeFormatting(string text)
+        {
+            text = text.Replace("|", Environment.NewLine);
+            text = text.Replace("[br]", Environment.NewLine);
+            text = text.Replace("<br>", Environment.NewLine);
+            text = text.Replace("<br/>", Environment.NewLine);
+            text = text.Replace("<br />", Environment.NewLine);
+            text = text.Replace("{\\i1}", "<i>");
+            text = text.Replace("{\\i0}", "</i>");
+            text = text.Replace("{\\i}", "</i>");
+            text = text.Replace("{\\b1}", "<b>'");
+            text = text.Replace("{\\b0}", "</b>");
+            text = text.Replace("{\\b}", "</b>");
+            text = text.Replace("{\\u1}", "<u>");
+            text = text.Replace("{\\u0}", "</u>");
+            text = text.Replace("{\\u}", "</u>");
+            return text;
+        }
+
+        public string EncodeFormatting(string text)

```

Comment:
I'm not really sure about this does UnknownSubtitle4 uses  `{\\i1}` to format italic? If yes then free to merge :)  /cc @niksedk